build-aux/pkg.nix

```diff
@@ -139,6 +139,7 @@ in stdenv.mkDerivation (finalAttrs: {
     touch source/build-aux/rust_boilerplate.mk
   '';
 
+  # TODO: Remove this, and test whether it is needed according to nix package sile-clang
```

Comment:
I tried the clang build (which worked out of the gate) both with Lua 5.2 and LuaJIT and both seemed to work both with and without this `preBuild` patch. Do you have any background on why it was added in the first place?

Reply:
Someone else wrote the commit, but it all happened in https://github.com/NixOS/nixpkgs/pull/77738 according to `git blame` on Nixpkgs.